### PR TITLE
Feature us2

### PR DESCRIPTION
**Pull request by Pia0912**
- backend (create add function, post endpoint; add uuid class , hobbyWithoutID class, write necessary tests)

- frontend (add add form, edit styling)

### CODE REVIEW

File: backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java [hunk 1]
```diff
@@ -22,5 +21,10 @@ public List<Hobby> listHobbies() {
         return this.hobbyService.list();
     }
 
+    @PostMapping
+    public List<Hobby> addParty(@RequestBody HobbyWithoutID newHobby){
```
Comment (sahedw) [inline backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java:25]: Is Party intentional? Would've guessed addHobby or something like that :D but you have more foresight into your project

**Review by sahedw — APPROVED** [review]
Really good job, Pia 🙌🏼

File: backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java (hunk 1 above)
Comment (Pia0912) [reply backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java:25]: Woopsie - You definetly did not see that haha 🤣

Comment (sahedw) [reply backend/src/main/java/de/neuefische/capstone/pia/backend/controller/HobbyController.java:25]: 😂

**Pull request merged**